packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -281,7 +281,7 @@ class IconButton extends StatelessWidget {
   /// {@macro flutter.material.RawMaterialButton.mouseCursor}
   ///
   /// Defaults to [SystemMouseCursors.click].
```

Comment:
Now the default depends on onPressed

Reply:
Will this work?

```dart
  /// {@macro flutter.material.RawMaterialButton.mouseCursor}
  ///
  /// If set to null, will default to
  /// - [SystemMouseCursors.forbidden], if [onPressed] is null
  /// - [SystemMouseCursors.click], otherwise
  final MouseCursor? mouseCursor;
```

Reply:
Sure